app/commands/solution/update_num_loc.rb

```diff
@@ -3,11 +3,32 @@ class Solution::UpdateNumLoc
 
   initialize_with :solution
 
-  def call = solution.update!(num_loc:)
+  def call
+    if iteration && iteration.num_loc.nil?
+      calc_job = Iteration::CalculateLinesOfCode.defer(iteration)
+      Solution::UpdateNumLoc.defer(prereq_jobs: [calc_job])
+      return
```

Comment:
Is this more or less readable?

```suggestion
      return Solution::UpdateNumLoc.defer(prereq_jobs: [
        Iteration::CalculateLinesOfCode.defer(iteration)
      ])
```